Makefile

```diff
@@ -147,3 +151,27 @@ docs/pykanidm/serve:
 	cd pykanidm && \
 	poetry install && \
 	poetry run mkdocs serve
+
+########################################################################
+
+release/kanidm: ## Build the Kanidm CLI
+	cargo build -p kanidm_tools --bin kanidm --release
```

Comment:
If we are using this, we still should indicate in the comments that a build profile ushould be set?

Reply:
So, to clarify, for each make target, should I replace: 

```shell
release/kanidm: ## Build the Kanidm CLI
cargo build -p kanidm_tools --bin kanidm --release
```

with

```shell
release/kanidm: ## Build the Kanidm CLI - ensure you include an environment variable KANIDM_BUILD_PROFILE
cargo build -p kanidm_tools --bin kanidm --release
```

?

Reply:
yes! :)